subsys/bluetooth/controller/Kconfig

```diff
@@ -86,4 +86,39 @@ choice SOFTDEVICE_CONTROLLER_VARIANT
 
 endchoice
 
+if BT_CTLR_ECDH
+choice BT_CTLR_ECDH_LIB
+	prompt "SoftDevice Controller ECDH library"
+
+config BT_CTLR_ECDH_LIB_OBERON
+	select NRF_OBERON
+	bool "Use nrf_oberon SW crypto library for ECDH"
```

Comment:
This becomes very long:
![image](https://user-images.githubusercontent.com/9802583/119883417-d0a87080-bf2f-11eb-81d6-70decc887c63.png)

I simply propose `nRF Oberon` here:
```suggestion
	bool "nRF Oberon (SW)"
```

Which looks much nicer, imho:
![image](https://user-images.githubusercontent.com/9802583/119884169-a0150680-bf30-11eb-8bcd-fa8bb9cf56db.png)